isthmus/src/main/java/io/substrait/isthmus/TypeConverter.java

```diff
@@ -56,6 +58,17 @@ public NamedStruct toNamedStruct(RelDataType type) {
     return NamedStruct.of(names, struct);
   }
 
+  public NamedStruct toNamedStruct(Map<String, RelDataType> nameToTypeMap) {
+    var names = Lists.<String>newArrayList();
+    var types = Lists.<Type>newArrayList();
```

Comment:
Can you use the ArrayList constructor directly?
```java
var names = new ArrayList<String>
```

The docs for the method state
```
Note: this method is now unnecessary and should be treated as deprecated. Instead, use the ArrayList [constructor](https://docs.oracle.com/javase/9/docs/api/java/util/ArrayList.html?is-external=true#%3Cinit%3E()) directly, taking advantage of ["diamond" syntax](http://goo.gl/iz2Wi).
```

Reply:
Changed